stage1/init/init.go

```diff
@@ -171,6 +172,50 @@ func machinedRegister() bool {
 	return found == 2
 }
 
+func lookupPath(bin string, paths string) (string, error) {
+	pathsArr := strings.Split(paths, ":")
+	for _, path := range pathsArr {
+		binPath := filepath.Join(path, bin)
+		binAbsPath, err := filepath.Abs(binPath)
+		if err != nil {
+			return "", fmt.Errorf("unable to find absolute path for %s", binPath)
+		}
+		if _, err := os.Stat(binAbsPath); err == nil {
+			return binAbsPath, nil
+		}
+	}
+	return "", fmt.Errorf("unable to find %q in %q", bin, paths)
+}
+
+func installAssets() error {
+	systemctlBin, err := lookupPath("systemctl", os.Getenv("PATH"))
+	if err != nil {
+		return err
+	}
+	bashBin, err := lookupPath("bash", os.Getenv("PATH"))
+	if err != nil {
+		return err
+	}
+	systemdShutdownBin, err := lookupPath("systemd-shutdown", "/lib/systemd:/usr/lib/systemd")
```

Comment:
I was mentioning earlier about trickiness of finding the systemd-shutdown and systemd, because I'm simply wondering if it those files are always in either `/lib/systemd` or `/usr/lib/systemd`. For example, doesn't Debian have those fancy, multi-archy `/lib/<triplet-foo>/systemd` we should care about?